evap/evaluation/management/commands/refresh_results_cache.py

```diff
@@ -11,11 +13,11 @@ class Command(BaseCommand):
     help = 'Clears the cache and pre-warms it with the results of all courses'
 
     def handle(self, *args, **options):
-        print("Clearing cache...")
+        logger.info("Clearing cache...")
         cache.clear()
 
-        print("Calculating results for all courses...")
+        logger.info("Calculating results for all courses...")
         for course in Course.objects.all():
             calculate_results(course)
 
-        print("Done.")
+        logger.info("Done with updating cache.")
```

Comment:
please make sure this gets printed on the console as well when calling this command